public/app/features/dashboard-scene/settings/variables/VariableEditorForm.tsx

```diff
@@ -22,9 +23,16 @@ interface VariableEditorFormProps {
   variable: SceneVariable;
   onTypeChange: (type: EditableVariableType) => void;
   onGoBack: () => void;
+  onDelete: (variableName: string) => void;
 }
 
-export function VariableEditorForm({ variable, onTypeChange, onGoBack }: VariableEditorFormProps) {
+export function VariableEditorForm({
+  variable,
+  onTypeChange,
+  onGoBack,
+  onDelete: propsOnDelete,
```

Comment:
I think we don't need to rename this callback since there is no other callback with the same name 😄 
```suggestion
  onDelete,
```